src/app/project/ontology/property-info/property-info.component.html

```diff
@@ -52,7 +52,7 @@
                 <mat-icon>tune</mat-icon>
             </button>
             <span
-                [matTooltip]="(resClasses.length > 0 ? 'The property can\'t be removed because it\'s in use' : 'Remove property from resource class')">
+                [matTooltip]="((resClasses.length > 0 || !propCanBeDeleted) ? 'The property can\'t be removed because it\'s in use' : 'Remove property from resource class')">
```

Comment:
I think this logic needs to be refactored. I can delete the property even though it says I can't. AS a side note, I think the delete icon should be the trash can for consistency reasons. The icon currently being used is normally used for "backspace".

<img width="437" alt="Screen Shot 2021-06-21 at 11 04 00" src="https://user-images.githubusercontent.com/60604010/122737279-2cd98880-d281-11eb-9078-887e67cf5c22.png">

Reply:
resolved in 5db5353